### PR TITLE
[#1733] improvement(docs): Add docs about partitioning, bucketing and sort ordering for Trino connector

### PR DESCRIPTION
**Pull request by yuqi1129**
### What changes were proposed in this pull request?

Add some description about partitioning, bucketing and sort ordering for Hive catalog and Iceberg catalog in Trino connector

### Why are the changes needed?

It's needed by users who use the Gravitino Trino connector. 

Fix: #1733 

### Does this PR introduce _any_ user-facing change?

N/A.

### How was this patch tested?

N/A.

### CODE REVIEW

**Comment by yuqi1129** [comment]
@diqiu50 
Could you please help me view the document?

File: docs/trino-connector/catalog-hive.md [hunk 1]
```diff
@@ -55,44 +76,68 @@ Support for the following alter table operations:
...
-| Property | Description                     | Default Value | Required | Since Version |
-|----------|---------------------------------|---------------|----------|---------------|
-| location | HDFS location for table storage | (none)        | No       | 0.2.0         |
+The following tables are the properties supported by the Hive table:
+
+| Property       | Description                             | Default Value                                              | Required | Since Version |
+|----------------|-----------------------------------------|------------------------------------------------------------|----------|---------------|
+| format         | Hive storage format for the table       | TEXTFILE                                                   | No       | 0.2.0         |
+| total_size     | Total size of the table                 | (none)                                                     | No       | 0.2.0         |
+| num_files      | Number of files                         | 0                                                          | No       | 0.2.0         |
```
Comment (diqiu50) [inline docs/trino-connector/catalog-hive.md:129]: Is these attributes like `total_size` and `num_files` can be set by the user?

File: docs/trino-connector/catalog-iceberg.md [hunk 2]
```diff
@@ -120,6 +165,9 @@ CREATE TABLE  "test.iceberg_test".database_01.table_01
 (
 name varchar,
 salary int
+) with (
```
Comment (diqiu50) [inline docs/trino-connector/catalog-iceberg.md:168]: Capitalized

File: docs/trino-connector/catalog-hive.md [hunk 3]
```diff
@@ -36,15 +36,36 @@ per catalog:
...
+
+### Create a schema 
+
+Users can create a schema with properties through Gravitino Trino connector as follows:
+
+```SQL
+CREATE SCHEMA "metalake.catalog".schema_name 
+```
+
+## Table operation
```
Comment (diqiu50) [inline docs/trino-connector/catalog-hive.md:50]: operations

File: docs/trino-connector/catalog-hive.md [hunk 4]
```diff
@@ -55,44 +76,68 @@ Support for the following alter table operations:
...
 statement.
 
+
+### Create a schema with properties
+
+Users can use the following example to create a schema with properties: 
+
+```sql
+CREATE SCHEMA "metalake.catalog".dbname
+with (
```
Comment (diqiu50) [inline docs/trino-connector/catalog-hive.md:96]: Capitalized

File: docs/trino-connector/catalog-hive.md (hunk 1 above)
Comment (yuqi1129) [reply docs/trino-connector/catalog-hive.md:129]: I don't think so, the word `The following tables are the properties supported by the Hive table` does not specify whether users can set it.

**Comment by yuqi1129** [comment]
@diqiu50 
Could you please assist me in reviewing it again? thanks

**Review by diqiu50 — APPROVED** [review]
LGTM

**Pull request merged**